hoot-js/src/main/cpp/hoot/js/schema/JsonOsmSchemaLoader.cpp

```diff
@@ -65,7 +66,22 @@ void JsonOsmSchemaLoader::load(QString path, OsmSchema& s)
   if (_baseDir.size() == 0)
   {
     s.update();
+
```

Comment:
@mingjiesu Can you just call Schema checker here? E.g.

````
SchemaChecker sc(s);
sc.checkUnknownVertexType();
sc.checkEmptyGeometry();
````

Or possibly add another function to check both. E.g.

````
SchemaChecker(s).check();
````